__tests__/app.test.js

```diff
@@ -54,4 +52,40 @@ describe('GET /api/topics', () => {
             })
         })
     })
+});
+describe('GET /api/articles', () => {
+    test("GET /api/articles/:article_id returns status 200 and an object to client with author, title, article_id, body, topic, created_at, votes, and article_img_url when given an article id", () => {
+        return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then((response) => {
+            const body = response.body
+            expect(body.article).toEqual({
```

Comment:
You could use a more flexible jest matchers such as **toMatchObject** or **objectContaining** keep the tests flexible if the test data was to change or extend in properties in the future. 💡